src/dndbeyond/content-scripts/character.js

```diff
@@ -1131,11 +1155,20 @@ async function rollAction(paneClass, force_to_hit_only = false, force_damages_on
             character.getSetting("warlock-hexblade-curse", false))
             critical_limit = 19;
         // Polearm master bonus attack using the other end of the polearm is considered a melee attack.
-        if (action_name.includes("Polearm Master") && (character.hasClassFeature("Great Weapon Fighting", true) || character.hasFeat("Great Weapon Fighting", true))) {
-            damages[0] = damages[0].replace(/[0-9]*d[0-9]+/g, "$&ro<=2");
+        if (action_name.includes("Polearm Master") && 
+            ((character.hasClassFeature("Great Weapon Fighting", true) || character.hasFeat("Great Weapon Fighting", true)) && 
+            (!character.hasClassFeature("Great Weapon Fighting 2024", true) || !character.hasFeat("Great Weapon Fighting 2024", true)))) {
+            damages[0] = damages[0].replace(/[0-9]*d[0-9]+/g, "$&min3");
+        }
+
+        if (action_name.includes("Pole Strike") && (character.hasFeat("Polearm Master 2024", true)) && 
```

Comment:
Can you have 2014 Polearm Master alongside 2024 Great Weapon Fighting?
If yes, then I see a bug, since you check for 2014 PM with 2014 GWF, and 2024 PM with 2024 GWF.

I'd suggest:
```js
const isPolearm = action_name.includes("Polearm Master") || (action_name.includes("Pole Strike") && character.hasFeat("Polearm Master 2024", true));

if (isPolearm && 
    (character.hasClassFeature("Great Weapon Fighting") || 
      character.hasFeat("Great Weapon Fighting"))) {
            damages[0] = damages[0].replace(/[0-9]*d[0-9]+/g, "$&ro<=2");
} else if (isPolearm &&
    (character.hasClassFeature("Great Weapon Fighting 2024") || 
     character.hasFeat("Great Weapon Fighting 2024"))) {
            damages[0] = damages[0].replace(/([0-9]*)d([0-9]+)([^\s+-]*)(.*)/g, (match, amount, faces, roll_mods, mods) => {
                return new Array(parseInt(amount) || 1).fill(`1d${faces}${roll_mods}min3`).join(" + ") + mods;
            });
}
```